### PR TITLE
No domain

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #4155 

This is a fork of #4155

### CODE REVIEW

**Comment by jasonLaster** [comment]
Sorry @hughugsy i had to create a new PR because your PR was based off of master and i couldn't update it.

The tests should pass now, so just waiting on @codehag's review.

**Comment by hughugsy** [comment]
No problem :+1:

File: package.json [hunk 1]
```diff
@@ -30,9 +30,9 @@
     "lint-fix": "yarn lint-js -- --fix",
     "mochi":
       "mochii --mc ./firefox --default-test-path devtools/client/debugger/new",
-    "mochid": "yarn mochi -- --jsdebugger",
-    "mochir": "yarn mochi -- --repeat 10",
-    "mochih": "yarn mochi -- --setenv MOZ_HEADLESS=1",
+    "mochid": "yarn mochi -- --jsdebugger --",
```
Comment (codehag) [inline package.json]: why do we need the two dashes at the end?

File: src/utils/sources-tree/addToTree.js [hunk 2]
```diff
@@ -107,7 +107,7 @@ export function addToTree(
   source: SourceRecord,
   debuggeeUrl: string
 ) {
-  const url = getURL(source.get("url"));
+  const url = getURL(source.get("url"), debuggeeUrl);
```
Comment (codehag) [inline src/utils/sources-tree/addToTree.js]: we might not need to do this anymore

File: src/utils/sources-tree/getURL.js [hunk 3]
```diff
@@ -60,7 +64,7 @@ export function getURL(sourceUrl: string): { path: string, group: string } {
         // with a weird URL. Just group them all under an anonymous group.
         return merge(def, {
           path: url,
-          group: "(no domain)",
+          group: defaultDomain,
```
Comment (codehag) [inline src/utils/sources-tree/getURL.js]: since this was fixed by devtools-html/devtools-core#698 we might not need this change. @jasonLaster thoughts?

Comment (jasonLaster) [reply src/utils/sources-tree/getURL.js]: @codehag i still think it is a good practice to scope URLs. For instance, an eval w/ sourceURL, which is in one of the pages.

Comment (codehag) [reply src/utils/sources-tree/getURL.js]: @jasonLaster cool, this is good to go from my side then

the `data` protocol issue can be investigated separately

File: package.json (hunk 1 above)
Comment (jasonLaster) [reply package.json]: i'll move this to a separate PR, but the dashes keep the file passed in from being in quotes.

**Comment by jasonLaster** [comment]
so... i don't know what to do w/ this test. it passes locally.

I'll look at it tomorrow mornign

**Comment by codehag** [comment]
Looking into it as well!

**Comment by jasonLaster** [comment]
Thanks @codehag!

**Pull request merged**